SU-SWS/decanter
# 72 main nav

# READY FOR REVIEW

# Summary
- Create Main site navigation component

# Needed By (Date)
- ASAP - Event calendar is already using this

# Urgency
- Event calendar is already using this

# Steps to Test

1. Pull this branch
1. Run grunt styleguide
2. Check that the main nav behaves as specified in the Figma document

# Affected Projects or Products
- Decanter

# Associated Issues and/or People
- https://github.com/SU-SWS/decanter/issues/72
